x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/action_connector_form/connector_form.tsx

```diff
@@ -47,6 +50,12 @@ interface Props {
   onFormModifiedChange?: (isModified: boolean) => void;
   setResetForm?: (value: ResetForm) => void;
 }
+
+interface ProviderConfig {
```

Comment:
nit: rename it to something that makes it clear that it's only used/needed by the inference connector, something like `InferenceConnectorProviderConfig`

Reply:
Updated in 82c3ef5aa019c0af2de3e8ddc8a9a0bcd51f5712